src/utils/issuanceModules.test.ts

```diff
@@ -24,49 +27,84 @@ import {
 
 import { getIssuanceModule } from './issuanceModules'
 
-describe('getIssuanceModule() - Mainnet', () => {
-  test('returns basic issuance module for any other index', async () => {
-    const expectedAddress = BasicIssuanceModuleAddress
+describe('getIssuanceModule() - Mainnet - IndexProtocol', () => {
```

Comment:
I don't love this naming convention but I think the alternative is a lot more nesting of describe blocks which is the greater evil.